/**
 * Copyright (c) 2014-present, The osquery authors
 *
 * This source code is licensed as defined by the LICENSE file found in the
 * root directory of this source tree.
 *
 * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
 */

syntax = "proto3";

package containerd.events;

message NamespaceCreate {
	string name = 1;
	map<string, string> labels  = 2;
}

message NamespaceUpdate {
	string name = 1;
	map<string, string> labels  = 2;
}

message NamespaceDelete {
	string name = 1;
}